docs/source/changelog.rst

```diff
@@ -17,6 +17,48 @@ All notable changes to this project will be documented in this file.
 
 We have listed major changes for each release below.
 
+3.3.0
+==============================================================================
+
+- **Release date:** 20th December 2019
+- **Downloads:** `Source downloads are available on GitHub <https://github.com/uccser/cs-field-guide/releases/tag/3.3.0>`__
+
+**Summary of changes:**
+
+This release adds a new chapter on 'Big Data', various improvements for interactives, and updated project and chapter icons.
+A configuration tool for the sorting boxes interactive has been added, allowing teachers to setup specific examples for testing.
+
+**Changelog:**
+
+- Add new chapter: Big Data.
+- Update project icon.
+- Update chapter icons to better reflect their topics.
+- Update rendering of some mathemetical equations.
+- Add missing glossary terms. `#1017 <https://github.com/uccser/cs-field-guide/issues/1017>`__
+- Add ability for resulting equation in matrix-simplifier to be copied and pasted into all versions of the scene-editor interactive. `#1168 <https://github.com/uccser/cs-field-guide/pull/1168>`__
+- Add ability to remove all equations in the matrix-simplifier interactive at once. `#1168 <https://github.com/uccser/cs-field-guide/pull/1168>`__
+- Fix spelling in title of regular expression search interactive. `#1172 <https://github.com/uccser/cs-field-guide/issues/1172>`__
+- Update layout, simplify difficult password, and hide plaintext passwords of password guesser interactive. `#1172 <https://github.com/uccser/cs-field-guide/issues/1172>`__
+- Improve sorting boxes interactive with clearer feedback and configurator for teachers. `#1196 <https://github.com/uccser/cs-field-guide/pull/1196>`__
+- Update number memory interactive to reuse shorter password if the user remembers the longer one. `#1172 <https://github.com/uccser/cs-field-guide/issues/1172>`__
+- Fix bug in colour matcher where some bits did not flip when clicked on. `#1167 <https://github.com/uccser/cs-field-guide/issues/1166>`__
+- Set external links to open in a new tab.  `#1175 <https://github.com/uccser/cs-field-guide/pull/1175>`__
```

Comment:
I think this overstates what we've done; only external links defined in the html files have this, links in the content (i.e. markdown) are unchanged. I think there's a pr in Verto related to this, will investigate in the new year